plugins/nordic-hid/fu-nordic-hid-cfg-channel.c

```diff
@@ -468,8 +469,18 @@ fu_nordic_hid_cfg_channel_add_peers(FuNordicHidCfgChannel *self, GError **error)
 		/* prohibit to close parent's communication descriptor */
 		fu_device_add_internal_flag(FU_DEVICE(peer),
 					    FU_DEVICE_INTERNAL_FLAG_USE_PARENT_FOR_OPEN);
-		/* probe&setup are the part of adding child */
 		fu_device_add_child(FU_DEVICE(self), FU_DEVICE(peer));
+
+		/* Remove child that does not support config channel DFU. */
+		if (!fu_device_probe(FU_DEVICE(peer), &error_peer) ||
```

Comment:
Doesn't setup call probe automatically?

Reply:
It does. Changed.

Initially I called it explicitly, because in `fu_device_setup` there is the following comment right above `fu_device_probe` call: `/* should have already been called */`